recipes-security/optee-imx/optee-test_3.10.0.imx.bb

```diff
@@ -60,4 +60,4 @@ do_install () {
 
 FILES:${PN} = "/usr/bin/ /lib*/optee_armtz/"
 
-COMPATIBLE_MACHINE = "(mx6-nxp-bsp|mx7-nxp-bsp|mx8-nxp-bsp)"
+COMPATIBLE_MACHINE = "(imx-nxp-bsp)"
```

Comment:
We have `uboot-imx` on generic-bsp, but we have `optee` on nxp-bsp. Is there a reason?

Reply:
Can it be used with mainline nowadays?

Reply:
It was restricted; so I kept it as well.